Add working directory related information to debugger guide (#2151)
ballerina-platform/ballerina-dev-website: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

Then, you can start a program, test, or remote debug session as shown below.

> >**Info:** The working directory will always be the package root if the user starts the debug session through VS Code. Users can use remote debugging for alternative working directories.

Reviewer: ```suggestion
>**Info:** For the debug sessions which get launched from the VS Code, the working directory will be the Ballerina package root. Users can use remote debugging for alternative working directories.
```

Reviewer: @praneesha Appreciate your input on this

Reviewer 2: Replied below.

> >**Info:** The working directory will always be the package root if the user starts the debug session through VS Code. Users can use remote debugging for alternative working directories.

Reviewer 2: ```suggestion
>**Info:** If you launch the debug session through VS Code, the working directory will always be the package root. However, you can use remote debugging for alternative working directories.
```

Reviewer: >

Reviewer: @praneesha How about `>**Info:** If you launch the debug session through VS Code, the working directory will be the Ballerina package root. However, you can use remote debugging for alternative working directories.`

Reviewer 2: +1

Author: Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2151/commits/9638a4849dbb2b96e940cfa945c9e12d58c00b23
